src/components/sq-select-search/sq-select-search.component.ts

```
@@ -220,6 +220,17 @@ export class SqSelectSearchComponent {
     this.nativeElement = element.nativeElement
   }
 
+  /**
+   * Lifecycle hook called when any input properties change.
+   *
+   * @param changes - The changes detected in the component's input properties.
+   */
+  async ngOnChanges(changes: SimpleChanges) {
+    if (this.open && changes.hasOwnProperty('options')) {
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)